src/main/java/com/example/tournamentMaker/excel/football/FootballLeagueWriterService.java

```diff
@@ -0,0 +1,187 @@
+
+            Font font;
+            for (String letter : firstLetterResults) {
+                font = switch (letter) {
+                    case "W" -> greenFont;
+                    case "L" -> redFont;
+                    case "D" -> blueFont;
+                    default -> throw new IllegalArgumentException("There are other letters in String than W,L,D");
+                };
+                richText.append(letter, (XSSFFont) font);
+                richText.append(" ");
+            }
+            cell.setCellValue(richText);
+
+            rowStart++;
+            colStart = 0;
+        }
+        footballStatisticWriterService.writeStatistic(workbook, sheet,
+                STATISTICS_ROW_START, STATISTICS_COLUMN_START, tournament);
+        return ExcelUtil.createExcelFile(filePath, workbook);
+    }
+
+    private void createColumnHeaders(int rowStart, int colStart, CellStyle cellStyle) {
+        Row row = sheet.createRow(rowStart);
+
+        Cell cell = row.createCell(colStart);
+        cell.setCellStyle(cellStyle);
+        sheet.setColumnWidth(colStart, COLUMN_WITH_NUMBER_EXCEL_WIDTH * ExcelUtil.COLUMN_WIDTH_UNIT);
+
+        colStart++;
+
+        cell = row.createCell(colStart);
+        cell.setCellValue("Team");
+        cell.setCellStyle(cellStyle);
+        sheet.setColumnWidth(colStart, TEAM_COLUMN_EXCEL_WIDTH * ExcelUtil.COLUMN_WIDTH_UNIT);
+
+        colStart++;
+
+        cell = row.createCell(colStart);
+        cell.setCellValue("MP");
+        cell.setCellStyle(cellStyle);
+        sheet.setColumnWidth(colStart, COLUMN_WITH_NUMBER_EXCEL_WIDTH * ExcelUtil.COLUMN_WIDTH_UNIT);
+
+        colStart++;
+
+        cell = row.createCell(colStart);
+        cell.setCellValue("W");
+        cell.setCellStyle(cellStyle);
+        sheet.setColumnWidth(colStart, COLUMN_WITH_NUMBER_EXCEL_WIDTH * ExcelUtil.COLUMN_WIDTH_UNIT);
+
+        colStart++;
+
+        cell = row.createCell(colStart);
+        cell.setCellValue("D");
+        cell.setCellStyle(cellStyle);
+        sheet.setColumnWidth(colStart, COLUMN_WITH_NUMBER_EXCEL_WIDTH * ExcelUtil.COLUMN_WIDTH_UNIT);
+
+        colStart++;
+
+        cell = row.createCell(colStart);
+        cell.setCellValue("L");
+        cell.setCellStyle(cellStyle);
+        sheet.setColumnWidth(colStart, COLUMN_WITH_NUMBER_EXCEL_WIDTH * ExcelUtil.COLUMN_WIDTH_UNIT);
+
+        colStart++;
+
+        cell = row.createCell(colStart);
+        cell.setCellValue("P");
+        cell.setCellStyle(cellStyle);
+        sheet.setColumnWidth(colStart, COLUMN_WITH_NUMBER_EXCEL_WIDTH * ExcelUtil.COLUMN_WIDTH_UNIT);
+
+        colStart++;
+
+        cell = row.createCell(colStart);
+        cell.setCellValue("Last 5");
+        cell.setCellStyle(cellStyle);
+        sheet.setColumnWidth(colStart, LAST_5_COLUMN_EXCEL_WIDTH * ExcelUtil.COLUMN_WIDTH_UNIT);
+    }
+
+    private CellStyle createCellStyle(Workbook workbook, IndexedColors color) {
```

Comment:
rownie dobrze mozesz od razu wyciagnac index zamiast przekazywaccaly duzy obiekt tylko po to zeyb z niego jeden prop wyciagnac